sandbox-web/src/main/java/com/exadel/sandbox/team5/OrderRestController.java

```diff
@@ -53,4 +59,22 @@ public OrderDto create(@RequestBody CreateOrder createOrder) {
         }
         return orderService.createOrder(createOrder);
     }
+
+    @ApiOperation("Generating QR code")
+    @GetMapping(value = "/validate/{uuid}", produces = MediaType.IMAGE_PNG_VALUE)
+    @ResponseStatus(HttpStatus.OK)
+    public byte[] generateQRCode(@PathVariable String uuid) {
```

Comment:
QR should be generated in ```     @PutMapping("/create") ``` in single transaction with order creation

Reply:
I moved QR to ```"/create"``` and changed method to @Post